pkgs/development/python-modules/pre-commit-hook-ensure-sops/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, buildPythonPackage, applyPatches, fetchFromGitHub, ruamel-yaml }:
```

One per line. `applyPatches` is unused.

```suggestion
{ lib
, buildPythonPackage
, fetchFromGitHub
, ruamel-yaml
}:
```